packages/timeline/src/ReactAxisGanttSeries.ts

```diff
@@ -0,0 +1,248 @@
+import { Axis } from "@hpcc-js/chart";
+import { Border2 } from "@hpcc-js/layout";
+import { ReactGantt } from "./ReactGantt";
+import { IAxisGanttData } from "./ReactAxisGantt";
+
+export class ReactAxisGanttSeries extends Border2 {
+
+    protected _topAxis: Axis = new Axis("origin")
+        .orientation("top")
+        .type("linear")
+        .shrinkToFit("none")
+        .overlapMode("hide")
+        .extend(0)
+        .tickFormat("d")
+        ;
+    protected _gantt: ReactGantt = new ReactGantt("origin")
+        .stroke("#000000")
+        .fitHeightToContent(true)
+        ;
+    protected _bottomAxis: Axis = new Axis("origin")
+        .orientation("bottom")
+        .type("linear")
+        .shrinkToFit("none")
+        .overlapMode("hide")
+        .extend(0)
+        .tickFormat("d")
+        ;
+    
+    protected _topAxisElement;
+    protected _contentElement;
+    protected _bottomAxisElement;
+    protected _topRect;
+    protected _contentRect;
+    protected _bottomRect;
+
+    protected rangeRenderer;
```

Comment:
Why is this not a regular method?

Reply:
I think it made sense to me in that moment to declare it in the constructor because it depends on a child widget, but there's no real reason it should be declared there.